.firebase/hosting.cHVibGlj.cache

```diff
@@ -0,0 +1,21 @@
+favicon.ico,1611096653197,8cbf926e6b19ff1ce6df71a2a54db8340895ac318343bdf4e8cd59b3a25eaa3a
+404.html,1611098126133,05cbc6f94d7a69ce2e29646eab13be2c884e61ba93e3094df5028866876d18b3
+images/customer.png,1611096653219,d49c9f75788823e2d8041b64b9265758533d8d0949445056b80ef273b048c5ff
+index.html,1611098126544,f4621c6473727ef7ecd05b80681e0a1bb5b21c61b606bc9a0561daf0ef88de67
+images/icon/android-chrome-192x192.png,1611096653220,90e49538ac1a10913ffb5bd8c104c53b519677cf9e74ca25d0a334fdbbbbbdd9
+images/icon/android-chrome-512x512.png,1611096653220,c8ea76b0f2bc5e6e83d144d3d533644501cb01e5ef8e2873e38b0350cb342844
+images/icon/apple-touch-icon.png,1611096653220,1c78ab76ad7064172373d1d5a1fabf350af88b622774ca1cb8365a5b48d752cd
+images/icon/favicon-16x16.png,1611096653222,ace369394a5df35692fde4df2d2309d66968d14e4654cc897c41d0e40255ff25
+images/icon/favicon-32x32.png,1611096653222,4dde64886476e6e726d401084707c78fa7b1a044e27e6a47aeed8064d459c5ac
+images/icon/mstile-150x150.png,1611096653222,3938980296734d96b57ee5be8ccd6dd66cd6b14f661882d1feccc7f52103b218
+images/icon/safari-pinned-tab.svg,1611096653222,1baccbeda476a82039fccb913c2e4744af4af00450285d9dfbcef5e781c1c340
+images/icon/banda.png,1611096653220,6cf6f8596e476e46a26c9a9e1f3491fa6328489d8204fdd041c654d01de8e620
+images/icon/banner_transparent.png,1611096653221,3ef7f3bbe258f04807492201b2d64fd76387cf7b83270a6f0e0fbcc6df102205
+images/icon/banner_white.png,1611096653221,a26d814283aa5d920b11dbf22cc4f212e345ac6af58f365b154c7f078c0943c9
+images/lunchmemo-bg.jpg,1611096653223,6d68867d49e0213b4f46ea7f9715eb75c5b352f6e3f36d691fd73dcf481387b4
+images/backgrounds/0.jpg,1611096653198,6d68867d49e0213b4f46ea7f9715eb75c5b352f6e3f36d691fd73dcf481387b4
+images/backgrounds/4.jpg,1611096653215,e279de7b87d55fedfb39db13b2b3ad7678e608b8003e32bd3877c9eb197f66c3
+images/backgrounds/1.jpg,1611096653201,cb272db63cdc4e712480a96bd740ee01916614feae19c22daf2c5e9ebe48b1d2
+images/backgrounds/2.jpg,1611096653206,668c13e42024e025e5fb56fd38a92f9887399480d4b0dc80a017bf6bce53f653
+images/backgrounds/5.jpg,1611096653219,aafb630fd06600cd8869e320534ae79371785436ac71082059b989b0232ed53c
+images/backgrounds/3.jpg,1611096653212,8361593d772923a40cb9a935cc8a2d2c0834b7c676b4844f746682322bb9d1fd
```

Comment:
Please add this file to the .gitignore!  Probably just a .firebase/*.cache would suffice